cpp/src/gcylon/utils/construct.hpp

```diff
@@ -20,23 +20,59 @@
 #include <cudf/io/types.hpp>
 #include <cuda.h>
 
+#include <stdlib.h>
+#include <time.h>
+
 using namespace std;
 
+/**
+ * construct a long sorted column with sequentially increasing values
+ * @param size
+ * @param value_start
+ * @return
+ */
+std::unique_ptr<cudf::column> constructLongColumn(int64_t size, int64_t value_start = 0, int step = 1) {
+    int64_t * cpu_buf = new int64_t[size];
```

Comment:
shall we use malloc or std::vector here?

Reply:
done.